src/scss/layouts/_chooseGameSection.scss

```diff
@@ -1,5 +1,4 @@
 .chooseGameSection {
-  position: relative;
```

Comment:
A czemu ta linijka jest usuwana?

Reply:
wydawało mi się, że wcześniej Piotrek napisał, że ma być bez pozycjonowania. Możliwe że źle zrozumiałem, i usunąłem nie to co trzeba. Przywrócić?

Reply:
wydaje mi się, że ja też dodałem tę linię, bo w moim stylu buttona dodałem position absolute. chciałem, żeby button był w pozycji absolute do parent elementu. potem dowiedziałem się, że to nie jest konieczne, dlatego ta linia zniknęła :)